### DISCUSSION TITLE
Description组件渲染数据问题

### DISCUSSION THREAD

**Discussion by johnwanzhi** [vbenjs/vue-vben-admin#1518] [Q&A]
我按照教程，使用我自己的API后，不渲染数据，是我的方式有问题吗？
![8XT1G)9~J @T~E@P2W UI1Q](https://user-images.githubusercontent.com/90312135/146945692-3017a484-229c-47e1-b482-9d2a86c983d6.png)
![PODHQXVFQ_G7R47EB18AE6O](https://user-images.githubusercontent.com/90312135/146945707-32f32b48-c9a6-4312-84fa-4ac7f66a8c39.png)

**Comment by Luyulin8** [comment]
兄弟，你解决了吗，我也遇到同样的问题了
